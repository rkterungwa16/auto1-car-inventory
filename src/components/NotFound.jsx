import React from 'react'
import { Link } from 'react-router-dom'

import logo from '../assets/logo.png'

const NotFound = () => (
  <div className='not-found__container'>
    <div className='not-found'>
      <img
        className='not-found__logo'
        src={logo} alt='Auto1'
      />
      <h2 className='not-found__text--dark'>
      404 - Not Found
      </h2>
      <p
        className='not-found__text--medium not-found__text--dark'
      >Sorry, the page you are looking for does not exist.</p>
      <p
        className='not-found__text--medium'
      >
    You can always go back to the
        <Link
          className='not-found__link' to='/'
        > homepage</Link>
      </p>
    </div>
  </div>
)

export default NotFound
