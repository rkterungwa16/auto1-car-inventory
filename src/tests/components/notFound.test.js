import React from 'react'
import Enzyme from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

import NotFound from '../../components/NotFound'

Enzyme.configure({ adapter: new Adapter() })
const { shallow } = Enzyme

test('should render NotFound correctly', () => {
  const wrapper = shallow(<NotFound />)
  expect(wrapper).toMatchSnapshot()
})
