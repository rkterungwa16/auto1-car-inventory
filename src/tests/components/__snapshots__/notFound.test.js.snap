// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFound correctly 1`] = `
<Fragment>
  <Header />
  <div
    className="not-found__container"
  >
    <div
      className="not-found"
    >
      <img
        alt="Auto1"
        className="not-found__logo"
        src="test-file-stub"
      />
      <h2
        className="not-found__text--dark not-found__text-extra-large"
      >
        404 - Not Found
      </h2>
      <p
        className="not-found__text--medium not-found__text--dark"
      >
        Sorry, the page you are looking for does not exist.
      </p>
      <p
        className="not-found__text--medium"
      >
        You can always go back to the
        <Link
          className="not-found__link"
          replace={false}
          to="/"
        >
           homepage
        </Link>
      </p>
    </div>
  </div>
</Fragment>
`;
